library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -1087,13 +1089,17 @@ public void onResponse(Call<FlagResponse> call, Response<FlagResponse> response)
 
             @Override
             public void onFailure(Call<FlagResponse> call, Throwable t) {
-                callback.onError(t.getLocalizedMessage(), -1);
+                ErrorResponse response = new Gson().fromJson(t.getLocalizedMessage(), ErrorResponse.class);
```

Comment:
can you use the static GSON class for JSON de/ser?

Reply:
I see.

Reply:
I meant `GsonConverter.Gson()`